src/WrapExternStages.cpp

```diff
@@ -145,6 +145,10 @@ class WrapExternStages : public IRMutator2 {
 }
 
 void wrap_legacy_extern_stages(Module m) {
+    if (!m.target().has_feature(Target::LegacyBufferWrappers)) {
```

Comment:
This removes the ability to call extern stages that request the old buffer_t (the uses_old_buffer_t argument to define_extern). Bailing out here would make it a mysterious error. We could assert that the uses_old_buffer_t argument is false unless this target flag is on to push people to upgrade their extern stages, or we could just continue to wrap those stages - those wrappers currently only get emitted if you explicitly request one, so (unlike the global wrapper for the pipeline code) they're harmless in terms of code size.

Reply:
Maybe in WrapExternStages::visit(const Call *) - that's when you discover if a call node has the uses_old_buffer_t flag.

Reply:
yep, testing now. (fwiw, it looks like we may not have *any* tests of `define_extern(use_old_buffer_t=true)` in Halide at the moment.)

Reply:
Correction: generator_aot_old_buffer_t tests this case.